JS files/catalog.js

```diff
@@ -0,0 +1,65 @@
+document.addEventListener("DOMContentLoaded", function () {
+    fetch("./recipes.json")
+        .then((response) => {
+            if (!response.ok) {
+                throw new Error("Network response was not ok " + response.statusText);
+            }
+            return response.json();
+        })
+        .then((recipes) => {
+            console.log("Fetched recipes:", recipes);
+            const catalogContainer = document.getElementById("catalog-container");
+            if (!catalogContainer) {
+                console.error("Catalog container not found");
+                return;
+            }
+
+            recipes.forEach((recipe) => {
+                const recipeElement = document.createElement("div");
+                recipeElement.className = "pro";
+                recipeElement.innerHTML = `
+                    <div class="image-container">
+                        <img src="${recipe.image}" alt="${recipe.name}" />
+                        <div class="overlay-text">${recipe.description}</div>
+                    </div>
+                    <div class="description">
+                        <span><b>${recipe.name}</b></span>
+                        <h2></h2>
+                        <div class="tags_div">
+                            <div class="tag">${recipe.time}</div>
+                        </div>
+                        <div class="review" style="color: rgb(243, 181, 25)">
+                            ${generateStars(recipe.reviews.user_reviews)}
+                        </div>
+                    </div>
+                    <a href="recipe.html?id=${recipe.id}">
+                        <i class="fas fa-shopping-basket basket"></i>

```

Comment:
Please just change this icon to a heart. There's too much gap between the cards so if you could do something to fix that it'd be great too. Nice work @Keerthivasan-Venkitajalam :)

Reply:
Will do that :) 👍🏻